Makefile

```diff
@@ -130,16 +130,13 @@ test:
 	@echo
 	go test -tags "${NOTARY_BUILDTAGS}" $(TESTOPTS) $(PKGS)
 
-test-full: TESTOPTS =
```

Comment:
Any reason for removing this target?  Admittedly, I don't use it much because it doesn't include `fmt` and `misspell`